packages/patternlab/styleguide/source/assets/js/modules/organizationNavigation.js

```diff
@@ -96,58 +154,113 @@ export default (function (window, document, $, undefined) {
       let $otherMenus = $('.ma__organization-navigation__subitems').not($thisMenu);
       let menuHeight = $menuWrapper.outerHeight();
 
-      $buttonParent.on('mouseenter mouseleave', function () {
-        let windowWidth = $(window).width();
-
-        if (windowWidth > mobileBreak) {
-          $('.section-toggle').remove();
-          $buttonParent.toggleClass('item-open');
-          $thisMenu.css('top', menuHeight);
-        }
-        else {
-          return false;
-        };
+      // $buttonParent.on('mouseenter mouseleave', function () {
+      $button.on("click", function (e) {
+        toggleMenu ();
+
+        // When focus moves to another menu item, close the open menu container.
+        // Only one menu container opens at a time.
+        $(".ma__organization-navigation__items .has-subnav").each(function () {
+          let $orgNavMenuButton = $(this).find(".subnav-toggle");
+
+          if ($(this).hasClass(classSubNavOpen)) {
+            if($(this).find(":focus").length === 0){
+              $(this).removeClass(classSubNavOpen);
+              $orgNavMenuButton.attr("aria-expanded", "false");
+              $(this).find(".ma__organization-navigation__subitems").attr("aria-hidden", "true");
+              $(this).find(".ma__organization-navigation__subitems").removeAttr("style");
+            }
+          }
+        });
       });
 
-      $button.on('focus', function () {
-        $thisMenu.find("a[href]").attr("tabindex", -1);
-        $otherMenus.find("a[href]").attr("tabindex", -1);
-
-        $button.keyup(function (e) {
+      // Close container with ESC.
+      $("body").on("keyup", function(e) {
+        if (e.key === "Escape") {
+          // Close submenu with ESC
+          // Set focus on its parent menubutton.
+          if ($buttonParent.find(":focus").length !== 0) {
+            closeMenuTasks();
+            $button.focus();
+          }
 
-          $('.item-open').removeClass('item-open');
+          // TO DO: Close mobile menu container.
```

Comment:
**Note for TO DO:**
This is tricky one.  It's meant to close the mobile menu container, which is different from one above.  The one above at L. 182 closes an open submenu.  Based on the way the mobile nav works, this is triggered when submenus are closed, which means one of the navigation buttons/links or the search is supposed to have focus.